central/processlisteningonport/store/postgres/full_store.go

```diff
@@ -142,7 +144,7 @@ func (s *fullStoreImpl) readRows(
 			},
 			DeploymentId:  msg.GetDeploymentId(),
 			PodId:         podID,
-			PodUid:        procMsg.GetPodUid(),
+			PodUid:        podUID,
```

Comment:
@JoukoVirtanen Could you please elaborate on the difference between `podID` and `podUID`?

Reply:
`podID` is the name of the pod and `podUID` is the UID of the pod. This confusing naming convention was borrowed from elsewhere. This is not just a problem with listening endpoints/PLOP. It should be fixed at some point. I don't think that the PRs I have open are the right place for that.

Reply:
As you obviously figured out, I found it confusing, hence seeked for clarification.